tests/Doctrine/Tests/ORM/Functional/SchemaTool/CompanySchemaTest.php

```diff
@@ -49,21 +48,4 @@ public function testSingleTableInheritance(Schema $schema): void
         self::assertFalse($table->getColumn('pricePerHour')->getNotnull());
         self::assertFalse($table->getColumn('maxPrice')->getNotnull());
     }
-
-    /**
-     * @group DBAL-115
-     */
-    public function testDropPartSchemaWithForeignKeys(): void
-    {
-        if (! $this->_em->getConnection()->getDatabasePlatform()->supportsForeignKeyConstraints()) {
-            self::markTestSkipped('Foreign Key test');
-        }
-
-        $sql = $this->_schemaTool->getDropSchemaSQL(
-            [
-                $this->_em->getClassMetadata(CompanyManager::class),
-            ]
-        );
-        self::assertCount(4, $sql);
```

Comment:
The SQL that the DBAL uses for managing schema is an implementation detail that may and will change.